### DISCUSSION TITLE
A Bitcoin Design Lightning Ring of Fire

### DISCUSSION THREAD

**Discussion by GBKS** [BitcoinDesign/Meta#106] [Ideas]
Quite the title... so as you may know, every Lightning node needs to connect to other Lightning nodes to create a network that money can flow through. Unlike Bitcoin nodes, which automatically find other nodes, Lightning nodes need to get manually connected to peers. So how do find these other nodes to connect to? You can use a node directory (like [1ml](https://1ml.com/node)), or you can connect to your friends nodes. That's where the "[Ring of Fire](https://github.com/Czino/ring-of-fire)" comes in, which describes such a network of cooperating node "partners".

The idea here is to create such a ring within members of this community to help each other get started with our Lightning nodes. To follow the idea [as described](https://github.com/Czino/ring-of-fire), we'd need to plan this thoroughly and create an actual ring of nodes in a specific order, etc. Alternatively, we could simplify that part and each of us just list our node addresses and we open up channels with each other a bit more loosely. This might be easiest for starters, and we can still upgrade to this ring approach later.

According to [this post](https://community.getumbrel.com/t/opening-channels-requests/66), it is recommended to open channels of around 500K to 1M sats, which is $275 to $550.

So to start out, is anyone interested in doing this? If so, should we just connect to each other loosely or form a tighter ring?

**Comment by Bosch-0** [comment]
I'm keen, I'll share my node once I get some more liquidity in it, its quite empty rn !

**Comment by vindard** [comment]
I'm down for joining as well. Anywhere better 500k - 1M is good with me (preference for 1M)

**Reply by GBKS** [reply]
Is exchanging node info here on Github problematic due to privacy? Maybe exchange in Slack instead? What's your preference.
